---a/net/minecraft/world/biome/BiomeGenRiver.java
+++b/net/minecraft/world/biome/BiomeGenRiver.java
@@ -1,13 +1,12 @@
 package net.minecraft.world.biome;
 
-public class BiomeGenRiver extends BiomeGenBase
-{
+public class BiomeGenRiver extends BiomeGenBase {
+
     private static final String __OBFID = "CL_00000181";
 
-    public BiomeGenRiver(int p_i1987_1_)
-    {
+    public BiomeGenRiver(int p_i1987_1_) {
         super(p_i1987_1_);
         this.spawnableCreatureList.clear();
     }
-}
+}
